tools/map/tests/test_build_tile_grid_province_records.py

```
@@ -8,20 +8,32 @@
         self.assertFalse(any("직할" in record["displayName"] for record in chinese))
         self.assertTrue(all(record["nameCh"] for record in chinese))
-        counties = [record for record in chinese if record["kind"] == "COUNTY"]
-        self.assertTrue(counties)
-        self.assertTrue(all(record["cityIndex"] is not None for record in counties))
+        self.assertTrue(all(record["kind"] == "SPATIAL_PROVINCE" for record in chinese))
+        self.assertTrue(all(record["jurisdictionId"] in jurisdictions for record in chinese))
+        chinese_jurisdictions = {
+            jurisdictions[record["jurisdictionId"]]["id"] for record in chinese
+        }
+        self.assertTrue(chinese_jurisdictions)
+        self.assertTrue(all(jurisdictions[id]["displayName"] for id in chinese_jurisdictions))
```

Ruff (0.16.3)

[error] Variable `id` is shadowing a Python builtin

(A001)